### ISSUE TITLE
Conjur can be run with verbose logging

### ISSUE THREAD

**Issue by jvanderhoof** [cyberark/conjur#698]
As someone running Conjur, I want to enable verbose logging, so that I can troubleshoot any issues which arise.

**Comment by orenbm** [comment]
@rafis3 this is relevant to what we talked about last week. If we decide to split the `debug` level to `debug` and `verbose` we can do this.

**Comment by rafis3** [comment]
@orenbm yes, I agree. But to support the capability that this issue is raising, we also need to support a better way to change the log level without restarting the container.

**Comment by szh** [comment]
This is implemented with log levels
